apps/webcomponents/src/app/components/gn-dataset-view-map/gn-dataset-view-map.component.ts

```diff
@@ -18,10 +22,22 @@ import { BaseComponent } from '../base.component'
   providers: [SearchFacade],
 })
 export class GnDatasetViewMapComponent extends BaseComponent implements OnInit {
-  constructor(injector: Injector, private mdViewFacade: MdViewFacade) {
+  constructor(
+    injector: Injector,
+    private mdViewFacade: MdViewFacade,
+    private changeDetector: ChangeDetectorRef
+  ) {
     super(injector)
   }
-  ngOnInit(): void {
-    this.mdViewFacade.loadFull('ee965118-2416-4d48-b07e-bbc696f002c2')
+  @Input() datasetId: string
+  link: DatasetDistribution
+  async init() {
+    super.init()
+    this.mdViewFacade.loadFull(this.datasetId)
```

Comment:
I don't think we should rely on the `mdViewFacade` here, as it won't be available in a web component. The `datasetId` should be passed to the web component as an attribute/input like for the chart and table. This can be tested locally.

Reply:
It is already passed as a datasetId, and I tried to remove this line and the webcomponent does not work anymore :frowning_face:

Reply:
Were you able to test this on your side with a docker container? It did not work on my side.